node/THIRD_PARTY_LICENSES_NODE

```diff
@@ -35785,7 +35785,7 @@ SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 ----
 
-Package: eslint-module-utils:2.11.0
+Package: eslint-module-utils:2.8.2
```

Comment:
pretty sure we need an approval for this, but I'm not sure
@barshaul

Reply:
you don't need to manually change it, it will be changed after merged with the daily ort workflow

Reply:
its a mistake, had a conflict and took the local

Reply:
Critical or anyhow itll be overridden tonight?
@barshaul